### PR TITLE
AM2R: Add Unlocked save station option.

### PR DESCRIPTION
**Pull request by Miepee**
Planning to later also do another configuration change for am2r (exclude certain doors in labs from being shuffled), not sure how i should handle preset migrations there. Should I just stuff the next thing also into v70? or should i make a v71 for it?

### CODE REVIEW

File: randovania/gui/ui_files/preset_am2r_patches.ui [hunk 1]
```diff
@@ -201,6 +201,20 @@
...
+            <widget class="QCheckBox" name="blue_save_doors_check">
+             <property name="text">
+              <string>Force blue Save Station Doors</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QLabel" name="blue_save_doors_label">
+             <property name="text">
+              <string>Ensures all Save Station doors are blue dors, even with door lock rando enabled.</string>
```
Comment (XenoWars) [inline randovania/gui/ui_files/preset_am2r_patches.ui:214]: dors typo, should be doors.

**Review by XenoWars — CHANGES_REQUESTED** [review]

File: randovania/gui/ui_files/preset_am2r_patches.ui [hunk 2]
```diff
@@ -201,6 +201,20 @@
...
+            <widget class="QCheckBox" name="blue_save_doors_check">
+             <property name="text">
+              <string>Force blue Save Station Doors</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QLabel" name="blue_save_doors_label">
+             <property name="text">
+              <string>Ensures all Save Station doors are blue doors, even with door lock rando enabled.</string>
```
Comment (ThanatosGit) [inline randovania/gui/ui_files/preset_am2r_patches.ui:214]: I'm fine if you want to call it internally "Blue door" but AM2R has more than one door in the color of blue AFAIK.
Also it is called "Normal door" in the Door lock settings. If calling it blue door is not already ambiguous, it is at least questionable when calling it by a different name in another setting. 
The same applies to the changelog.

**Review by ThanatosGit — CHANGES_REQUESTED** [review]

File: randovania/gui/ui_files/preset_am2r_patches.ui (hunk 2 above)
Comment (Miepee) [reply randovania/gui/ui_files/preset_am2r_patches.ui:214]: whoops, yeah, that shouldnt happen. thanks.

Comment (Miepee) [reply randovania/gui/ui_files/preset_am2r_patches.ui:214]: Altho I just noticed that Prime has the same issue xP
grafik
grafik
But that's up to change in a later PR i guess.

**Review by ThanatosGit — APPROVED** [review]

**Pull request merged**